elements/pfe-accordion/BaseAccordionHeader.scss

```diff
@@ -0,0 +1,113 @@
+:host {
+  --pf-c-accordion__toggle--Color: var(--pf-global--Color--100, #151515);
```

Comment:
@markcaron do you know if these vars are meant to be user accessible? Is it ok if the user's preferences for these values from higher up the DOM tree are ignored?

Reply:
every instance of properties set on :host should be removed and set as fallbacks